tech/languages/python/python-installation.md

```diff
@@ -14,8 +14,8 @@ Python 3 is already pre-installed on Fedora. Let's use it!
 
 ## Running Python
 
-1. Open the _Terminal_ (press `Alt` + `F1`, type _Terminal_ and click on the black square icon or just press `Enter`).
-2. To run Python 3 type `python3`. You should see something like this:
+1. Open your _terminal_. (In GNOME, press `Alt` + `F1`, type _Terminal_ and press <kbd>Enter</kbd>.)
```

Comment:
If we're to guide <developers> how to run a terminal, it should be a standalone guide, for various desktop spins. I don't find it fitting for Python-installation / running python guide.

I'd like to assume everyone knows how to run terminal and a command in it. Otherwise half of the developer portal would need a makeover :) .

Reply:
This has been hare already, the change does not make it nay worse.

Reply:
Doing nothing is not better than doing something to make it better ;). Or do you want me to simply remove it (I can open a PR)?

Reply:
I like the note as proposed. We can share the page with GNOME/Fedora/Linux beginners and whoever knows how to opena  terminal simply skips it.

Reply:
> If we're to guide how to run a terminal, it should be a standalone guide, for various desktop spins.

Sounds good; I'll happily link to it instead, once it's up!

But before then, as Miro says, I'd like to make Python developers coming from another OS feel welcome :)

Reply:
> I like the note as proposed. We can share the page with GNOME/Fedora/Linux beginners and whoever knows how to opena terminal simply skips it.

Linux beginner + Python beginner does not look like our target audience to me :), but I may be wrong. Python is a first language for many developers, so feel free to keep it if you like it!

But I definitely would not make it a general rule/guidance for Developer Portal.